js-exercises/abbreviate-string/abbreviateString.js

```diff
@@ -0,0 +1,20 @@
+const isError = str => typeof str !== 'string';
```

Comment:
This itself is not an error. This is merely a check. So name accordingly. Also, `isError` should tell whether an object is of the Error type or not.